doc/components/EventComponent.js

```diff
@@ -24,7 +24,7 @@ EventComponent.Prototype = function() {
           .on('click', this.onClick)
           .append($$('span').addClass('se-name').append(node.name)),
         $$('div').addClass('se-source').append(
-          $$('strong').append(this.i18n.t('event')),
+          $$('strong').append(this.getLabel('event')),
```

Comment:
You now need to provide the labels via addLabel. You can do this in doc/DocumentationReaderConfig.js just add some lines 

```js
config.addLabel('event', 'Event'); // we don't need multi language here
```